### DISCUSSION TITLE
Perte de débit mkv en mel

### DISCUSSION THREAD

**Discussion by mutchu** [quietvoid/dovi_tool#319] [Q&A]
Bonjour,
Pourquoi j’ai une légère perte de débit sur un film en Mkv (mel) en le convertissant en profil 8 ? Je pensais que ça ce produisait uniquement sur les film en FEL…
Merci

**Comment by quietvoid** [comment]
Please use English.

You did not provide enough detail for me to help in any way.
Provide more info on the commands you used.
